### PR TITLE
Update SDK to 2.12

### PR DESCRIPTION
**Pull request by polina-c**
Update SDK version to 2.12(null safe) and add // @dart=2.9 to all files that do not respect null safety yet.

### CODE REVIEW

File: packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart [hunk 1]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart:1]: please change this to 2021, as that's the year this file was added

File: packages/devtools_app/lib/src/inspector/layout_explorer/layout_explorer.dart [hunk 2]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/inspector/layout_explorer/layout_explorer.dart:1]: 2019

File: packages/devtools_app/lib/src/inspector/layout_explorer/ui/widget_constraints.dart [hunk 3]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/inspector/layout_explorer/ui/widget_constraints.dart:1]: 2019

File: packages/devtools_app/lib/src/primitives/geometry.dart [hunk 4]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/primitives/geometry.dart:1]: 2019

File: packages/devtools_app/lib/src/shared/console_service.dart [hunk 5]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/shared/console_service.dart:1]: 2021

File: packages/devtools_app/lib/src/ui/filter.dart [hunk 6]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/ui/filter.dart:1]: 2020

File: packages/devtools_app/lib/src/ui/label.dart [hunk 7]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/ui/label.dart:1]: 2019

File: packages/devtools_app/lib/src/ui/tab.dart [hunk 8]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/ui/tab.dart:1]: 2021

File: packages/devtools_app/test/auto_complete_test.dart [hunk 9]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/auto_complete_test.dart:1]: 2021

File: packages/devtools_app/test/search_test.dart [hunk 10]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/search_test.dart]: 2021

File: packages/devtools_app/test/table_test.dart [hunk 11]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/table_test.dart]: 2019

File: packages/devtools_app/test/test_utils/app_size_test_controller.dart [hunk 12]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/test_utils/app_size_test_controller.dart]: 2020

File: packages/devtools_app/test/vm_service_private_test.dart [hunk 13]
```diff
@@ -1,3 +1,9 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/vm_service_private_test.dart]: 2020

File: packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/lib/src/inspector/inspector_breadcrumbs.dart:1]: changed locally

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**